### DISCUSSION TITLE
How can I improve my about page?

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#60538] [Programming Help]
### Body

The link is https://linknprk.github.io/about.html

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by OshekharO** [comment]
Use popular css frameworks such as bootstrap and also use custom css wherever required. For styling you can checkout my repositories as I am mostly in web dev so you may find something useful.

Mark this as correct answer if it satisfy you.

**Comment by Beargruug** [comment]
i would also suggest to use a css framework. (https://ant.design/ , https://mui.com/) are very easy to use for that. Just import the components and you're ready to go.

**Comment by AliAlmasi** [comment]
[TailwindCSS](https://tailwindcss.com/docs/) is also a good and easy-to-use CSS library

**Comment by queenofcorgis** [comment]
@OshekharO, @L9us, @AliAlmasi thanks for all the awesome suggestions. Let us know @linknprk what you ended up using and any learnings. Share the knowledge 💡
